projects/vdk-core/src/vdk/internal/builtin_plugins/ingestion/ingester_router.py

```diff
@@ -102,6 +92,7 @@ def send_object_for_ingestion(
                     f"Currently possible values are {list(self._ingester_builders.keys())}",
                 )
             )
+        return ingester
```

Comment:
For once I agree with codacy that this is an issue.

Reply:
Do we have tests for this class?

Reply:
Yes. 
 - [test_ingester_router](https://github.com/vmware/versatile-data-kit/blob/634febe37cab75ffc36cd88ef186b7c910b8e03c/projects/vdk-core/tests/vdk/internal/builtin_plugins/ingestion/test_ingester_router.py#L16)
 - [functional ingest tests](https://github.com/vmware/versatile-data-kit/tree/634febe37cab75ffc36cd88ef186b7c910b8e03c/projects/vdk-core/tests/functional/ingestion)